conans/test/unittests/model/options_test.py

```diff
@@ -163,7 +163,7 @@ def test_basic(self):
 class TestOptionsNone:
     @pytest.fixture(autouse=True)
     def _setup(self):
-        options = {"static": [None, 1, 2], "other": "ANY", "more": ["None", 1]}
+        options = {"static": [None, 1, 2], "other": [None, "ANY"], "more": ["None", 1]}
```

Comment:
This aligns with the behavior of settings, that need to declare this way to allow undefined value and any value. So far options were behaving differently and "ANY" allowed for None too.